src/page-weekly-snapshot/LocaleSummaryTable.tsx

```diff
@@ -0,0 +1,368 @@
+        <TableHeadingCell>
+          <TextContainerHeading>
+            <Right>{heading} </Right>
+            <LeftHeading>(per 100k)</LeftHeading>
+          </TextContainerHeading>
+        </TableHeadingCell>
+      </tr>
+      <BorderDiv />
+      <tr>
+        <TableCell>
+          Incarcerated {heading}
+          <HorizontalRule />
+        </TableCell>
+      </tr>
+      {incarceratedDataRow}
+      <BorderDiv />
+      <tr>
+        <TableCell>
+          Overall State {heading}
+          <HorizontalRule />
+        </TableCell>
+      </tr>
+      <tr>
+        <TableCell>
+          <TextContainer>
+            <Left>{formatThousands(stateData)}</Left>
+            <Right>
+              {formatOrdinal(stateRank)} lowest of {NUM_STATES}
+            </Right>
+          </TextContainer>
+        </TableCell>
+      </tr>
+    </>
+  );
+}
+
+const LocaleSummaryTable: React.FC<{
+  stateName: string | undefined;
+  stateNames: string[];
+}> = ({ stateName, stateNames }) => {
+  const localeState = useLocaleDataState();
+  const localeData = localeState.data;
+  const {
+    state: { facilities: facilitiesState },
+  } = useFacilities();
+  const facilities = Object.values(facilitiesState);
+
+  let casesPerCapita = 0;
+  let casesPerCapitaRank = 0;
+  let deathsPerCapita = 0;
+  let deathsPerCapitaRank = 0;
+
+  let incarceratedCasesPerCapita = 0;
+  let incarceratedDeathsPerCapita = 0;
+
+  let hasCaseData = true;
+  let hasDeathData = true;
+
+  const allStateMetrics = getAllStateData(localeData, stateNames);
+
+  // TODO: currently getting state name from the drop down
+  // will need to update whenever that changes
+  if (stateName) {
+    const selectedStateCasesRank = getStateRank(
+      allStateMetrics,
+      stateName,
+      "casesPerCapita",
+    );
+
+    casesPerCapita = selectedStateCasesRank?.[0];
+    casesPerCapitaRank = selectedStateCasesRank?.[1];
+
+    const selectedStateDeathsRank = getStateRank(
+      allStateMetrics,
+      stateName,
+      "deathsPerCapita",
+    );
+
+    deathsPerCapita = selectedStateDeathsRank?.[0];
+    deathsPerCapitaRank = selectedStateDeathsRank?.[1];
+
+    const totalIncarceratedCases = getTotalIncarceratedValue(
+      facilities,
+      "cases",
+    );
+    const totalIncarceratedDeaths = getTotalIncarceratedValue(
+      facilities,
+      "deaths",
+    );
+    const totalIncarceratedPopulation = getTotalIncarceratedValue(
+      facilities,
+      "population",
+    );
+    incarceratedCasesPerCapita = getPerCapita(
+      totalIncarceratedCases.incarceratedData,
+      totalIncarceratedPopulation.incarceratedData,
+    );
+    hasCaseData = totalIncarceratedCases.hasData;
+    incarceratedDeathsPerCapita = getPerCapita(
+      totalIncarceratedDeaths.incarceratedData,
+      totalIncarceratedPopulation.incarceratedData,
+    );
+    hasDeathData = totalIncarceratedDeaths.hasData;
+  }
+
+  return (
+    <>
+      <HorizontalRule width={"93%"} marginLeft={"20px"} />
```

Comment:
i see what you mean now about the magic numbers ... the margin value at least is something that could be turned into a constant (since it is used both here and by the `Column` component). Why do you even need the width value though? will it not just fill its container on auto width?

Reply:
After playing around with some different CSS ideas, I spoke to Juan directly about this and ended up deciding on a design closer to this, which was much more straightforward to implement: 
![image](https://user-images.githubusercontent.com/29155017/87476006-527afe00-c5eb-11ea-8217-16a74fc25cec.png)